content/22.vaccines-trad.md

```diff
@@ -0,0 +1,507 @@
+India is also developing vaccines using cutting-edge nucleic-acid-based platforms.
+These include ZyCov-D, a DNA vaccine produced by Zydus Cadila, HGCO19 and India’s first mRNA vaccine, produced by Genova and HDT Biotech Corporation (of the U.S.) [@url:https://www.bbc.com/news/world-asia-india-55748124].
+Additionally, in February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of an intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine called BBV154 [@url:https://www.bharatbiotech.com/intranasal-vaccine.html].
+A large number of Indians have been vaccinated with the AstraZeneca-University of Oxford vaccine, known as Covishield in India, which is also produced by the Serum Institute of India [@url:https://www.bbc.com/news/world-asia-india-55748124].
+
+In China, the Sinopharm-Beijing Institute vaccine, the Sinopharm-Wuhan Institute of Biological Products vaccine, the Sinovac Biotech (CoronaVac) vaccine, and CanSino Biologics vaccine are the main vaccines being distributed.
+The Sinopharm-Beijing vaccine has been distributed to at least 16 countries.<!--automate-->
+Indeed, Sinovac and Sinopharm have estimated that they will be able to produce 2 billion doses by the end of 2021, and they have been able to distribute vaccines as aid to the Philippines and Pakistan [@url:https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html].
+In contrast, the Sinopharm-Wuhan vaccine, which has been approved for use in China since February 25th, 2021, has been distributed almost exclusively within China, with limited supplies distributed to the United Arab Emirates [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#wuhan].<!--To Do: update & clean citation-->
+On the same date, the CanSino vaccine was approved for use in China and has been granted emergency use in Mexico and Pakistan, which were two participating countries in the CanSino phase 3 trials [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#cansino].
+
+However, the vaccine approval and distribution processes in China have come under increased scrutiny from other nations.
+China was criticized for administering vaccines to thousands of government officials and state-owned businesses in September 2020, prior to the completion of phase III clinical trials [@doi:10.1001/jama.2021.1961].
+The behavior of Chinese officials has also come into question due to misinformation campaigns questioning the safety of Western vaccine candidates such as Moderna and Pfizer-BioNTech in a way that is intended to highlight the benefits of their own vaccine candidates [@url:https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html].
```

Comment:
```suggestion
The behavior of Chinese officials has also come into question due to misinformation campaigns questioning the safety of Western vaccine candidates such as Moderna and Pfizer-BioNTech in a way that is intended to highlight the benefits of their own vaccine candidates [@url:https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html].
China in particular took aim at mRNA technologies, but Chinese companies have since developed their own mRNA vaccines targeting the Omicron variant, one of which is due to begin trials soon in the UAE [@url:https://www.reuters.com/business/healthcare-pharmaceuticals/chinese-omicron-specific-mrna-covid-vaccine-candidate-be-trialed-uae-2022-04-30/].
```